LEGO1/lego/sources/anim/legoanim.h

```diff
@@ -289,8 +289,16 @@ class LegoAnimNodeData : public LegoTreeNodeData {
 
 // SIZE 0x08
 struct LegoAnimActorEntry {
-	LegoChar* m_name;     // 0x00
-	undefined4 m_unk0x04; // 0x04
+	enum {
+		e_actorType2,
```

Comment:
enums begin with 0 implicitly. you need to assign a different initial value here

Reply:
Oh you are correct! Will fix.